src/nwb2bids/_converters/_dandi_utils.py

```diff
@@ -74,3 +77,64 @@ def _get_dataset_description_from_valid_dandiset_metadata(
 
     dataset_description = DatasetDescription(**dataset_description_kwargs)
     return dataset_description, messages
+
+
+def _get_dataset_description_from_invalid_dandiset_metadata(
+    raw_metadata: typing.Any,
+) -> tuple[DatasetDescription | None, list[InspectionResult]]:
+    dataset_description_kwargs = dict()
+
+    dandiset_identifier = raw_metadata.get("identifier", "??????")
+    dataset_description_kwargs["Name"] = raw_metadata.get("name", f"DANDI Archive Dandiset {dandiset_identifier}")
+    dataset_description_kwargs["BIDSVersion"] = "1.10"
+
+    assets_summary = raw_metadata.get("assetsSummary", dict())
+    data_standards = assets_summary.get("dataStandard", [])
+
+    messages = []
+    if any(data_standard.get("identifier", "") == _BIDS_RRID for data_standard in data_standards):
+        reason = (
+            "Dandiset is already organized to the BIDS standard. If only a partial conversion is desired, "
+            "please raise an issue on https://github.com/con/nwb2bids/issues/new to discuss the use case."
+        )
+        message = InspectionResult(
+            title="Dandiset is already BIDS",
+            reason=reason,
+            solution="Skip the conversion of this Dandiset.",
+            category=Category.INTERNAL_ERROR,
```

Comment:
The same issue with `Category.INTERNAL_ERROR`. To me an internal error usually means something is wrong with the `nwb2bids` application. The problem we are encountering has to do with the provided data. Maybe we need another category?